packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -30,7 +30,11 @@ class ComplianceRemediationButton extends React.Component {
     )
 
     rulesWithRemediations = (rules, system) => {
-        return rules.filter(rule => rule.remediationAvailable).map(
+        return rules.filter((rule) => (
+            rule.remediationAvailable &&
+            this.ruleProfile(rule, system).supported &&
+            rule.compliant === false
```

Comment:
Apparently it was possible before to try and remediate rules which passed :man_shrugging: